Sources/Sentry/include/SentrySession+Private.h

```diff
@@ -8,6 +8,8 @@ NSString *nameForSentrySessionStatus(SentrySessionStatus status);
 @interface
 SentrySession (Private)
```

Comment:
I made this refactoring elsewhere: change this from a category to a class extension (anonymous category) then you don't have to redeclare the property in SentrySession.m; instead, you can just `#include "SentrySession+Private.h"